### ISSUE TITLE
Commit Android code

### ISSUE THREAD

**Issue by jonkri** [hesa/coachassistant#57]

**Comment by hesa** [comment]
Have you merged a sub tree in git?

**Comment by jonkri** [comment]
No, unfortunately. But this page seems to outline the steps in a pretty clear way:

https://www.kernel.org/pub/software/scm/git/docs/howto/using-merge-subtree.html

Just let me know the Android repository if you want me to give it a go. :)
